### PR TITLE
Further operationalize the battle station

### PR DESCRIPTION
**Pull request by doctaphred**
Resolved 111 pyflakes violations and 13,762 pycodestyle violations.

(16,910 pycodestyle violations remain, of an initial 30,672.)

### CODE REVIEW

File: umami/gp.py [hunk 1]
```diff
@@ -56,14 +55,11 @@ def dist2(ls, x1, x2=None):
 def grad_dist2(ls, x1, x2=None):
     if x2 is None:
         x2 = x1
-        
+
     # Rescale.
     x1 = x1 / ls
     x2 = x2 / ls
-    
-    N = x1.shape[0]
```
Comment (llllllllll) [inline umami/gp.py:64]: these are used in the weave call below

Comment (doctaphred) [reply umami/gp.py:64]: Ah, dang. Good catch, I'll back these lines out tomorrow and add # noqa or something.

Comment (doctaphred) [reply umami/gp.py:64]: Fixed the mistake and squashed the commit.

Unfortunately pyflakes doesn't honor `# noqa`, so I'm now using flake8. It should be configurable to only check for certain errors in a build script, until we get them all ironed out.

Comment (llllllllll) [reply umami/gp.py:64]: @doctaphred I was thinking of using umami to tune a new model, is there more work needed for this PR?

**Pull request merged**